cloudinit/config/schema.py

```diff
@@ -34,7 +34,7 @@
 {property_doc}
 {examples}
 """
-SCHEMA_PROPERTY_TMPL = '{prefix}**{prop_name}:** ({type}) {description}'
+SCHEMA_PROPERTY_TMPL = '{prefix}**{prop_name}:** ({prop_type})'
```

Comment:
We have dropped the description of each property here in your commit 859bcc2f8ab371f2bfe19bacc4cd31430e50ece7. This makes both our online docs and the command-line docs a lot less helpful when people try to figure out how to use a given key.

This results in more sparse/terse documentation that no longer explains the intent and expectations of each property value for both commandline docs and online docs as we can see in the [write_files docs](https://cloudinit.readthedocs.io/en/latest/topics/modules.html#write-files)

```
content: (string) Optional content to write to the provided path. When content is present and encoding is not ‘text/plain’, decode the content prior to writing. Default: ‘’
```

but in your local branch we've truncated descriptions leaving us with the less helpful
```
content: (string)
```

we can also quick test these renderd docs locally on the commandline with
```
PYTHONPATH=. python3 -m cloudinit.cmd devel schema --docs all
```